app/jobs/import_url_job.rb

```diff
@@ -10,19 +10,23 @@ class ImportUrlJob < Hyrax::ApplicationJob
   queue_as Hyrax.config.ingest_queue_name
 
   before_enqueue do |job|
-    operation = job.arguments.last
     operation.pending_job(job)
   end
 
+  # Retrieves the operation for the job
+  def operation
+    arguments.reduce(:merge).fetch(:operation)
+  end
+
   # @param [FileSet] file_set
   # @param [Hyrax::BatchCreateOperation] operation
-  def perform(file_set, operation)
+  def perform(file_set:, operation:, headers: {})
```

Comment:
Is it possible to do this change without changing to keyword arguments?  I think the switch to keyword arguments would change the method signature and thus would be backwards incompatible.